charts/kubewarden-defaults/chart-values.yaml

```diff
@@ -62,6 +62,18 @@ policyServer:
   #     certs:
   #       - "cert4"
   sourceAuthorities:
+  security:
+    pod:
+      runAsNonRoot: true
+    container:
+      readOnlyRootFilesystem: true
+      privileged: false
+      runAsNonRoot: true
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+          - "all"
+
```

Comment:
I see these are the prior default values we set. I don't think we will change them anytime soon, but I would prefer to have them defined via code inside of the controller (see my comment on the other PR).

I would propose to leave this entire section commented, to be a guideline to the user in case he wants to change it. At the same, by being commented, the default values inside of the `.go` files would take precedence.

Long story short, I fear that if we change these values in the go code, but forget to update this section of the helm chart, the change won't be enforced to our users.

Reply:
This is a good point, I agree with it.

Reply:
Done